cob_hardware_config/common/upload_robot.launch

```diff
@@ -1,13 +1,34 @@
 <?xml version="1.0"?>
 <launch>
 
-    <arg name="robot" default="$(optenv ROBOT !!NO_ROBOT_SET!!)"/>
-    <arg name="pkg_hardware_config" default="$(find cob_hardware_config)"/>
+	<arg name="robot" default="$(optenv ROBOT !!NO_ROBOT_SET!!)"/>
+	<arg name="pkg_hardware_config" default="$(find cob_hardware_config)"/>
+	<arg name="robot_description" default="robot_description"/>
+	<arg name="config_path" default="$(find cob_hardware_config)/$(arg robot)/config"/>
```

Comment:
no need for this launch arg: we can simply ise the pkg_hardware_config as we do for all other entries in cob_bringup